pico_w/wifi/ota_update/README.md

```diff
@@ -1,16 +1,18 @@
 # Overview
 
-This example uses a python script running on a host machine to _push_ new operating software image to a specified Pico 2 W running this `ota_update` example image.  The incoming image is programmed into Pico 2 W flash memory.
+This example demonstrates how to implement an Over The Air (OTA) software update mechanism using facilities provided by the RP2350 bootrom.
+
+ A python script runs on a host machine to _push_ a new operating software image to a Pico 2 W running the `ota_update` example image. The incoming image is received via the LwIP IP stack and programmed into Pico 2 W flash memory.
```

Comment:
Small nitpick, but I'd be inclined to change "operating software image" to just "software image", as "operating software" might be easily confused with "operating system"?